### ISSUE TITLE
Parse attributes on literal expressions

### ISSUE THREAD

**Issue by dtolnay** [dtolnay/syn#397]
```rust
extern crate syn;

fn main() {
    let code = "fn f() { let _ = #[noop] 0; }";
    println!("{:#?}", syn::parse_str::<syn::Item>(code).unwrap());
}
```

**Comment by dtolnay** [comment]
Affects [`attr-stmt-expr.rs`](https://github.com/rust-lang/rust/blob/7678d5021e5c984af52f4d1e9147f7a2d6a7ddc3/src/test/run-pass-fulldeps/proc-macro/attr-stmt-expr.rs).

**Comment by dtolnay** [comment]
Fixed in 5d314dcd9a02cbd070460e2ebd17c3abaf4c70d8.
